server/lib/validation/languages/ko.js

```diff
@@ -0,0 +1,54 @@
+// Minimum of characters that qualify as a sentence.
+const MIN_CHARACTERS = 1;
+
+// Maximum of characters allowed per sentence to keep recordings in a manageable duration.
+const MAX_CHARACTERS = 50;
+
+const INVALIDATIONS = [{
+  fn: (sentence) => {
+    // To properly tokenize Korean, We need some heavy tokenizers (ex: mecab-ko, nori, ...),
+    // For counting letters those tokenizers are not necessary.
+
+    // Before counting letters, normalization should be applied, since length may be tripled
+    const normalizedSenetence = sentence.normalize('NFC');
+    return normalizedSenetence.length < MIN_CHARACTERS || normalizedSenetence.length > MAX_CHARACTERS;
+  },
+  error: `Number of characters must be between ${MIN_CHARACTERS} and ${MAX_CHARACTERS} (inclusive)`,
+}, {
+  regex: /[0-9]+/,
+  error: 'Sentence should not contain numbers',
+}, {
+  regex: /[<>+*#@%^[\]()/]/,
+  error: 'Sentence should not contain symbols',
+}, {
+  regex: /[a-zA-Z]/,
+  error: 'Sentence should not contain latin alphabet characters',
+}, {
+  // One Korean letter is composed with two or three letters,
+  // in order of (consonant(1st) - vowel(2nd) - consonant(3rd, optional)).
+  // It should be not allowed to use them separately, since That could cause various pronounciation.
+  // This is only possible for "Hangul Syllables" (already composed letters in unicode)
+  regex: /[ㄱ-ㅎㅏ-ㅣ]/,
+  error: 'Sentence should not contain partial korean charaters',
+}, {
+  // Since there are so may kinds of "should not be allowd" letters,
+  // It would be convenient to allow only certain type of characters.
+  // examples: CJK chinese letters, Japanese letters, Korean specific chinese letters (aka hanja),
+  //           not-used symbols (semicolon, colon - native korean sentences do not contain them),
+  //           better to be excluded symbols (quote, tilda, ...),
+  //           characters that can be normalized into normal characters with NFKC normalization (ⓐ, ㈜, ...),
+  //           historical korean letters (aka 옛한글 - ㆆ, ㅿ, ㆁ, ...)
+  //           ...
+  /** @param {string} sentence */
+  fn: (sentence) => {
+    // With non-destructive normalization, We could allow users to input "Hangul Jamo" (not composed letters in unicode, aka 첫가끝).
+    // This is convenient for whom using specific type of keyboard layout (sebeolsik), since these layout inserts Hangul Jamo.
```

Comment:
minor:

```suggestion
    // This is convenient for contributors using a specific type of keyboard layout (sebeolsik), since this layout inserts Hangul Jamo.
```